pkg/apis/monitoring/v1/prometheus_types.go

```diff
@@ -381,6 +381,24 @@ type CommonPrometheusFields struct {
 	// TracingConfig configures tracing in Prometheus. This is an experimental feature, it may change in any upcoming release in a breaking way.
 	// +optional
 	TracingConfig *PrometheusTracingConfig `json:"tracingConfig,omitempty"`
+	// BodySizeLimit defines per-scrape on response body size.
+	// Only valid in Prometheus versions 2.45.0 and newer.
+	BodySizeLimit ByteSize `json:"bodySizeLimit,omitempty"`
```

Comment:
this should be a `*ByteSize` since it's an optional value

Reply:
done